sdk/storage/Azure.Storage.Blobs/src/Models/BlobTraits.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+
+namespace Azure.Storage.Blobs.Models
+{
+    /// <summary>
+    /// Specifies trait information to be included when listing blobs with the
+    /// <see cref="BlobContainerClient.GetBlobsAsync"/> and
+    /// <see cref="BlobContainerClient.GetBlobsByHierarchyAsync"/>
+    /// operations.
+    /// </summary>
+    [Flags]
+    public enum BlobTraits
+    {
+        /// <summary>
+        /// Default flag specifying that no flags are set in <see cref="BlobTraits"/>.
+        /// </summary>
+        None = 0,
+
+        /// <summary>
+        /// Flag specifying that metadata related to any current
+        /// or previous copy operation should be included.
+        /// </summary>
+        CopyOperationStatus = 1,
```

Comment:
We should shorten this to `CopyStatus` - that matches the actual property name that gets returned.